.gitignore

```diff
@@ -0,0 +1,6 @@
+.coverage*
```

Comment:
This is why `.coveragerc` wasn't added

Reply:
There is a way to fix this. You can explicitly _include_ something in the gitignore after it was excluded I think

Reply:
Oh? I also doesn't apply to anything which is already in Git, so I just shoved it in